templates/ro/access-default.txt

```diff
@@ -0,0 +1,27 @@
+Stimate domn/Stimată doamnă:
```

Comment:
Definitely makes sense, thanks for following-up.

I can use something like _În atenția dumneavoastră,_ which translates mot-a-mot _To your attention_ or _To whom it may concern_ , would this work?

Reply:
That sounds perfect :)